src/platform-includes/getting-started-install/javascript.remix.mdx

```diff
@@ -1,7 +1,24 @@
-```bash {tabTitle:npm}
-npm install --save @sentry/remix
-```
+We recommend installing the SDK through our installation wizard:
 
-```bash {tabTitle:Yarn}
-yarn add @sentry/remix
+```bash
+npx @sentry/wizard@latest -i remix
 ```
+
+The wizard will prompt you to log in to Sentry. It will then automatically do the following steps for you:
+
+- create two files in the root directory of your project, `entry.client.tsx` and `entry.server.tsx` (if they don't exist yet)
+- add the default Sentry.init() for the client and serverside SDK, in these files respectively
```

Comment:
This kind of makes it sound like there are separate SDKs for the client and server. Maybe this is better?

```suggestion
- add the default `Sentry.init()` for the client in `entry.client.tsx` and the server in `entry.server.tsx`
```